Client/Application/AppDelegate.swift

```diff
@@ -191,6 +191,8 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
          */
         Task {
             await FeatureManagement.fetchConfiguration()
+            // Ecosia: lifecycle tracking
```

Comment:
Can we maybe mention on the comment that this needs to happen after we fetch Unleash's toggles so they are added as context?

```suggestion
            // Ecosia: Lifecycle tracking. Needs to happen after Unleash start so that the flags are correctly added to the analytics context.
```